### ISSUE TITLE
KeyError: 'quicksettings'

### ISSUE THREAD

**Issue by sleel** [Linaqruf/sd-notebook-collection#29]
repo_type:AUTOMATIC1111

KeyError                                  Traceback (most recent call last)
[<ipython-input-3-93be8be172f7>](https://localhost:8080/#) in <cell line: 115>()
    113 config["show_progress_every_n_steps"] = 10
    114 config["show_progressbar"] = True
--> 115 if config["quicksettings"]:
    116     config["quicksettings"] = "sd_model_checkpoint, sd_vae, CLIP_stop_at_last_layers, use_old_karras_scheduler_sigmas, always_discard_next_to_last_sigma"
    117 else:

KeyError: 'quicksettings'

repo_type:Anapnoe    no such problem

**Comment by Ryuzhal** [comment]
same here

**Comment by Linaqruf** [comment]
> > It should have been fixed with this commit: [8a13dce](https://github.com/Linaqruf/sd-notebook-collection/commit/8a13dcea3da3a2fc00520d26a708ec43d3b7dac5). Please try it and let me know if there is something wrong.
> > Thank you.
> 
> It still doesn't fix the error

Can you confirm it still hasnt fixed? because there is no error in my side, thank you

**Comment by Linaqruf** [comment]
![image](https://github.com/Linaqruf/sd-notebook-collection/assets/50163983/b75ce6f3-f92e-42a2-ac7e-4e799163d49c)

**Comment by ReEnMikki** [comment]
> > > It should have been fixed with this commit: [8a13dce](https://github.com/Linaqruf/sd-notebook-collection/commit/8a13dcea3da3a2fc00520d26a708ec43d3b7dac5). Please try it and let me know if there is something wrong.
> > > Thank you.
> > 
> > 
> > It still doesn't fix the error
> 
> Can you confirm it still hasnt fixed? because there is no error in my side, thank you

Yeah, error is still happening for me
![image](https://github.com/Linaqruf/sd-notebook-collection/assets/114931925/fe29d214-3637-40f5-bc9b-93f08b1fadf9)
![image](https://github.com/Linaqruf/sd-notebook-collection/assets/114931925/64450292-b7d1-41d5-b5bf-3c4ee29e1677)

**Comment by Linaqruf** [comment]
Are you sure you're not using old notebook commit?

The new condition for `quicksettings` is : 
```python
if "quicksettings" in config:
    config["quicksettings"] = "sd_model_checkpoint, sd_vae, CLIP_stop_at_last_layers, use_old_karras_scheduler_sigmas, always_discard_next_to_last_sigma"
elif "quicksettings_list" in config:
    config["quicksettings_list"] = ["sd_model_checkpoint", "sd_vae", "CLIP_stop_at_last_layers", "use_old_karras_scheduler_sigmas", "always_discard_next_to_last_sigma"]
write_config(config_file, config)
```

While that line in your console logs was from https://github.com/Linaqruf/sd-notebook-collection/commit/a991e9b3e0764c5c933a8328424951d1bf043011
```python 
if config["quicksettings"]:
    config["quicksettings"] = "sd_model_checkpoint, sd_vae, CLIP_stop_at_last_layers, use_old_karras_scheduler_sigmas, always_discard_next_to_last_sigma"
else:
    config["quicksettings_list"] = [
    "sd_model_checkpoint", 
    "sd_vae", 
    "CLIP_stop_at_last_layers", 
    "use_old_karras_scheduler_sigmas", 
    "always_discard_next_to_last_sigma"
    ]
```

If you're using colab pro please reload the notebook so you can get the latest commit.

**Comment by ReEnMikki** [comment]
> Are you sure you're not using old notebook commit?
> 
> The new condition for `quicksettings` is :
> 
> ```python
> if "quicksettings" in config:
>     config["quicksettings"] = "sd_model_checkpoint, sd_vae, CLIP_stop_at_last_layers, use_old_karras_scheduler_sigmas, always_discard_next_to_last_sigma"
> elif "quicksettings_list" in config:
>     config["quicksettings_list"] = ["sd_model_checkpoint", "sd_vae", "CLIP_stop_at_last_layers", "use_old_karras_scheduler_sigmas", "always_discard_next_to_last_sigma"]
> write_config(config_file, config)
> ```
> 
> While that line in your console logs was from [a991e9b](https://github.com/Linaqruf/sd-notebook-collection/commit/a991e9b3e0764c5c933a8328424951d1bf043011)
> 
> ```python
> if config["quicksettings"]:
>     config["quicksettings"] = "sd_model_checkpoint, sd_vae, CLIP_stop_at_last_layers, use_old_karras_scheduler_sigmas, always_discard_next_to_last_sigma"
> else:
>     config["quicksettings_list"] = [
>     "sd_model_checkpoint", 
>     "sd_vae", 
>     "CLIP_stop_at_last_layers", 
>     "use_old_karras_scheduler_sigmas", 
>     "always_discard_next_to_last_sigma"
>     ]
> ```
> 
> If you're using colab pro please reload the notebook so you can get the latest commit.

Well, it's your official notebook link, and not an old copied one
![image](https://github.com/Linaqruf/sd-notebook-collection/assets/114931925/73c128a1-ea89-4566-84f7-7c392a489729)

**Comment by ReEnMikki** [comment]
Nevermind, I opened a new tab of this notebook and I see things change like the warning no longer being there. The problem was that while this notebook is still the official one, I kept the tab opening (ever since the problem was still in effect and the old commit still didn't work) and only tried running it again on that same tab of outdated notebook, instead of reloading it to refresh the page for the changes to take effect 🥲

**Comment by Linaqruf** [comment]
Yeah, sorry. I pushed so many commits today, and probably it got fixed while you were still viewing the last updated commit.
Also, sometimes there can be a delay in reflecting the changes after the commit has already been pushed to GitHub

**Comment by sleel** [comment]
> Yeah, sorry. I pushed so many commits today, and probably it got fixed while you were still viewing the last updated commit. Also, sometimes there can be a delay in reflecting the changes after the commit has already been pushed to GitHub

it works Thank you so much!
